cmd/dlv/dlv_test.go

```diff
@@ -507,3 +509,45 @@ func TestTypecheckRPC(t *testing.T) {
 		t.Errorf("%d errors", errcount)
 	}
 }
+
+// TestDap verifies that a dap server can be started and shut down.
+func TestDap(t *testing.T) {
+		const addr = "127.0.0.1:40575"
```

Comment:
`listenAddr` for consistency with other tests in the same file?

Reply:
Done